static/css/feed.css

```diff
@@ -0,0 +1,13 @@
+.icon{
+    padding-left:3px;
+    padding-right: 3px;
+    align-self: center;
+    align-content: center;
+}
+
+
+
+.col{
+    padding:0px !important;
```

Comment:
Did you try adding the p-0 class to the element instead of overriding existing bootstrap class styles?

Reply:
Also, I found that bootstrap provides a "no-gutters" class that you can add to the "row" element, which will remove all padding from columns belonging to that row.  That might be what you need.

Reply:
it wasn't actually doing anything.  It was residue from a previous version

Reply:
I'll remove it and submit another pull request